services/QuillLMS/app/models/classroom.rb

```diff
@@ -173,6 +173,22 @@ def grade_as_integer
     -1
   end
 
+  def provider_classroom?
+    google_classroom? || clever_classroom?
+  end
+
+  def clever_classroom?
+    clever_id.present?
+  end
+
+  def google_classroom?
+    google_classroom_id.present?
+  end
+
+  def provider_classroom
+    return 'Google Classroom' if google_classroom?
```

Comment:
Does this not have a Clever side since it's not used yet?

Reply:
Correct.  I have a separate Clever PR that will address this.